### ISSUE TITLE
ESRGAN results visualization 

### ISSUE THREAD

**Issue by AlexanderSlav** [eriklindernoren/PyTorch-GAN#89]
Why did you use default **mode='nearest**' in nn.functional.interpolate ? From my point of view, it will be more honest and clear using **'bicubic'** because it produces better quality.

`imgs_lr = nn.functional.interpolate(imgs_lr, scale_factor=4)
 img_grid = denormalize(torch.cat((imgs_lr, gen_hr), -1))
 save_image(img_grid, "images/training/%d.png" % batches_done, nrow=1, normalize=False)`

**Comment by datduong** [comment]
Hi, also in `denormalize` function, should tensor be multiplied with 255, because transforms.ToTensor() converts the pixels to 0-1 range?

**Comment by datduong** [comment]
Nvm. `save_image` already multiplies the pixel values with 255.
